ubo-common/src/main/resources/config/ubo-common/solr/main/solr-schema.json

```diff
@@ -1,745 +1,754 @@
+            "words": "lang/stopwords_en.txt",
+            "ignoreCase": "true"
+          },
+          {
+            "class": "solr.StopFilterFactory",
+            "format": "snowball",
+            "words": "lang/stopwords_fr.txt",
+            "ignoreCase": "true"
+          },
+          {
+            "class": "solr.GermanMinimalStemFilterFactory"
+          },
+          {
+            "class": "solr.EnglishMinimalStemFilterFactory"
+          }
+        ]
+      }
+    }
+  },
+  {
+    "add-field": {
+      "name": "abstract",
+      "type": "ubo_text",
+      "multiValued": true,
+      "indexed": true,
+      "stored": false
+    }
+  },
+  {
+    "add-field": {
+      "name": "ae_lsf",
+      "type": "ubo_id",
+      "multiValued": true,
+      "indexed": true,
+      "stored": false
+    }
+  },
+  {
+    "add-field": {
+      "name": "all",
+      "type": "ubo_text",
+      "multiValued": true,
+      "indexed": true,
+      "stored": false
+    }
+  },
+  {
+    "add-field": {
+      "name": "conference",
+      "type": "ubo_text",
+      "multiValued": true,
+      "indexed": true,
+      "stored": false
+    }
+  },
+  {
+    "add-field": {
+      "name": "dedup",
+      "type": "ubo_id",
+      "multiValued": true,
+      "indexed": true,
+      "stored": false
+    }
+  },
+  {
+    "add-field": {
+      "name": "facet_host_title",
+      "type": "string",
+      "multiValued": true,
+      "indexed": true,
+      "stored": false
+    }
+  },
+  {
+    "add-field": {
+      "name": "facet_topic",
+      "type": "string",
+      "multiValued": true,
+      "indexed": true,
+      "stored": false
+    }
+  },
+  {
+    "add-field": {
+      "name": "facet_person",
+      "type": "string",
+      "multiValued": true,
+      "indexed": true,
+      "stored": false
+    }
+  },
+  {
+    "add-field": {
+      "name": "facet_publisher",
+      "type": "string",
+      "multiValued": true,
+      "indexed": true,
+      "stored": false
+    }
+  },
+  {
+    "add-field": {
+      "name": "genre",
+      "type": "ubo_id",
+      "multiValued": false,
+      "indexed": true,
+      "stored": true
+    }
+  },
+  {
+    "add-field": {
+      "name": "host_genre",
+      "type": "ubo_id",
+      "multiValued": false,
+      "indexed": true,
+      "stored": true
+    }
+  },
+  {
+    "add-field": {
+      "name": "host_title",
+      "type": "ubo_text",
+      "multiValued": true,
+      "indexed": true,
+      "stored": true
+    }
+  },
+  {
+    "add-field": {
+      "name": "host_year",
+      "type": "pint",
+      "multiValued": false,
+      "indexed": true,
+      "stored": true
+    }
+  },
+  {
+    "add-field": {
+      "name": "importID",
+      "type": "string",
+      "multiValued": false,
+      "indexed": true,
+      "stored": false
+    }
+  },
+  {
+    "add-field": {
+      "name": "issue",
+      "type": "ubo_text",
+      "multiValued": false,
+      "indexed": false,
+      "stored": true
+    }
+  },
+  {
```

Comment:
sorry for large diff because of tab->whitespace changes
the only really new thing here is the new field "article_number"